internal/stolonctl/pkg/cluster/upgrade.go

```diff
@@ -39,6 +39,19 @@ const (
 )
 
 func Upgrade(ctx context.Context, config Config) error {
+	status, err := GetStatus(config)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	masterStatus, err := status.getMasterStatus()
+	if err != nil {
+		return trace.Wrap(err, "Cannot start upgrade.")
```

Comment:
Capitalizing and full sentences are only for logging - not the error messages:
```go
trace.Wrap(err, "cannot start upgrade")
```
but
```go
log.Warn("Cannot start upgrade.")
```